opentelemetry-api/src/opentelemetry/configuration/__init__.py

```diff
@@ -18,12 +18,42 @@
 """
 Simple configuration manager
 
-This is a configuration manager for the Tracer and Meter providers. It reads
-configuration from environment variables prefixed with
-``OPENTELEMETRY_PYTHON_``:
+This is a configuration manager for OpenTelemetry. It reads configuration
+values from environment variables prefixed with
+``OPENTELEMETRY_PYTHON_`` whose characters are only all caps and underscores.
+The first character after ``OPENTELEMETRY_PYTHON_`` must be an uppercase
```

Comment:
I think this sentence is not needed as the sentence before already implies it.

Reply:
Fixed.